src/postgraphile/http/createPostGraphileHttpRequestHandler.ts

```diff
@@ -413,7 +413,10 @@ export default function createPostGraphileHttpRequestHandler(
         )
       : null;
 
-    if (subscriptions) {
+    if (
+      subscriptions &&
+      (options.websocketOperations === 'all' || options.websocketOperations === 'subscriptions')
```

Comment:
Need to handle the current case without breaking. Really we want to make `websocketOperations` have a default; but this will suffice.

```suggestion
      (!options.websocketOperations || options.websocketOperations === 'all' || options.websocketOperations === 'subscriptions')
```

Reply:
Actually we can handle this in a much simpler way:

```suggestion
      options.websocketOperations !== 'none'
```